fluentcheck/classes/check.py

```diff
@@ -11,7 +11,7 @@ class Check:
 
     def _import_assertion_modules(self):
         ass = importlib.import_module(Check.ASSERTIONS_PACKAGE)
-        assertion_modules = list()
+        assertion_modules = []
```

Comment:
Out of curiosity, I am not aware of any improvements to using a literal syntax. Is there any literature or some reference I can read up to understand why the literal syntax is better?

Reply:
Hi, you can find plenty of sources talking about that Eg. https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/ 

In short: if you use constructors like dict(), list(), etc you get a performance penalty - even if a small one - on the standard C implementation of the python interpreter. This also seems more relevant for python 2 than python 3 cPython.

Anyway, not using literal constructors is in my coding style as I feel they're more readable but I see the point of this PR

The only thing that I would not include is the extra file added!

Reply:
Hi @csparpa :wave: 

Yes, using literal constructors improves the readability of the code.
There were more [issues](https://deepsource.io/gh/withshubh/fluentcheck/issues/?category=recommended) caught by DeepSource. I hope you had a look at them.

Reply:
Thanks for the link! Good stuff.